portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** A comment item. **/
+public final class  Comment {
+
+  private final long id;
+  private final String comment_text;
+  private final long timestamp;
+
+  public Comment(long id, String comment_text, long timestamp) {
```

Comment:
Java usually uses Camel-Case naming style. You can google "Camel-Case naming with Java" for more information. Please replace "comment_text" with "commentText".